src/components/TransactionItemRow/index.tsx

```diff
@@ -323,19 +321,7 @@ function TransactionItemRow({
             [CONST.SEARCH.TABLE_COLUMNS.EXPORTED]: (
                 <View
                     key={CONST.SEARCH.TABLE_COLUMNS.EXPORTED}
-                    style={[
-                        StyleUtils.getReportTableColumnStyles(
-                            CONST.SEARCH.TABLE_COLUMNS.EXPORTED,
-                            false,
-                            false,
-                            false,
-                            areAllOptionalColumnsHidden,
-                            false,
-                            false,
-                            false,
-                            isExportedColumnWide,
-                        ),
-                    ]}
+                    style={[StyleUtils.getReportTableColumnStyles(CONST.SEARCH.TABLE_COLUMNS.EXPORTED, false, false, false, false, false, isExportedColumnWide)]}
```

Comment:
`isExportedColumnWide` is the 8th argument.
```suggestion
                    style={[StyleUtils.getReportTableColumnStyles(CONST.SEARCH.TABLE_COLUMNS.EXPORTED, false, false, false, false, false, false, isExportedColumnWide)]}
```

Reply:
Nice catch!